### ISSUE TITLE
Ross to supply new Figure 3

### ISSUE THREAD

**Issue by rjsmethurst** [karenlmasters/gz-hubbleseq#3]

**Comment by karenlmasters** [comment]
Figure is there, but would help if Ross could write a caption.

**Comment by sandorkruk** [comment]
Has @RossHart been using his debiasing when selecting the subsamples and making this plot? The numbers in the ipython notebook and in the paper don't quite match up. I doubt this would change the plot significantly, but it would change the numbers.

**Comment by RossHart** [comment]
Hey, yes I used my new data to make that plot. I can share the data I used somewhere if that would be useful. As for the caption, it would read:

Fig. XX: (a) Galaxy Zoo winding score from Eq. XX vs. measured pitch angles from SpArcFiRe for all spirals with at least one reliably identified arc (see Hayes 14 and Hart et al. 17). (b) Galaxy Zoo bulge prominence from Eq. XX vs. SDSS $r$-band bulge luminosity as measured from Simard et al. 2011. The grey contours indicate where 20, 40, 60 and 80% of the galaxies lie in each plot and the dashed lines show the best fit straight line for each plot.

I ended up using:

w = 0 p_tight + 0.5 p_medium + 1.0 p_loose and 
B = 0 p_none + 0.2 p_just_noticable + 0.8 p_obvious + 1.0 p_dominant (as used in the original draft). Hope this all helps.
